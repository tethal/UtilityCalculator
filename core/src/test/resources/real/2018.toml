[general]
start_date = 2018-04-01
end_date = 2019-03-31
tenant = ["Marie Černá", "Jindřišská 16", "111 50 Praha 1"]
owner = ["Jan Novák", "majitel@example.com"]
report_place = "V Praze"
report_date = 2019-04-30
sources = ["Vyúčtování služeb od Společenství vlastníků za rok 2019"]

[[deposits]]
description="brezen-unor"
count=12
amount=4000

[[heating]]
annual_cost = 7790
start_date = 2018-01-01
end_date = 2018-12-31

[[heating]]
annual_cost = 8820
start_date = 2019-01-01
end_date = 2019-12-31

[[other_fees]]
start_date = 2018-01-01
end_date = 2018-12-31
annual_cost = 8592

[[other_fees]]
start_date = 2019-01-01
end_date = 2019-12-31
annual_cost = 9900

[[cold_water.reading]]
meter_id = "1"
reading_date = 2018-04-01
state = 55.786

[[cold_water.reading]]
meter_id = "1"
reading_date = 2019-04-01
state = 85.6


[[cold_water.tariff]]
start_date = 2018-01-01
end_date = 2018-12-31
unit_amount = 87.1342

[[cold_water.tariff]]
start_date = 2019-01-01
end_date = 2019-12-31
unit_amount = 89.66

[[hot_water.reading]]
meter_id = "1"
reading_date = 2019-04-01
state = 49.9

[[hot_water.reading]]
meter_id = "1"
reading_date = 2018-04-01
state = 36.452

[[hot_water.tariff]]
start_date = 2018-01-01
end_date = 2018-12-31
unit_amount = 82.673866

[[hot_water.tariff]]
start_date = 2019-01-01
end_date = 2019-12-31
unit_amount = 89.66

[[hot_water.heating_basic_cost]]
start_date = 2018-01-01
end_date = 2019-12-31
annual_cost = 1647.48

[[hot_water.heating_consumable_tariff]]
start_date = 2018-01-01
end_date = 2019-12-31
unit_amount = 137.195978
