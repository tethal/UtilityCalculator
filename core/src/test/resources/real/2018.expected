Report:
  dateRange: 2018-04-01...2019-04-01
  tenant: Marie Černá, Jindřišská 16, 111 50 Praha 1
  owner: Jan Novák, majitel@example.com
  placeDate: V Praze, 2019-04-30
  sources: Vyúčtování služeb od Společenství vlastníků za rok 2019
  sections:
    Studená voda: 2616.64
    Teplá voda: 4627.78
    Vytápění: 8294.70
    Ostatní poplatky: 8919.00
    Přijaté zálohy: -48000
    TOTAL: -23541.88

Studená voda:
  readings:
    [1] 2018-04-01...2019-04-01: 55.786...85.6 = 29.814
  priceList:
    2018-04-01...2019-01-01: 22.361 * 87.1342 = 1948.36
    2019-01-01...2019-04-01: 7.454 * 89.66 = 668.28

Teplá voda:
  readings:
    [1] 2018-04-01...2019-04-01: 36.452...49.9 = 13.448
  priceList:
    2018-04-01...2019-01-01: 10.086 * 82.673866 = 833.85
    2019-01-01...2019-04-01: 3.362 * 89.66 = 301.44
  heatingBasicParts:
    2018-04-01...2019-04-01: 12.00 * 137.29 = 1647.48
  heatingConsumableParts:
    2018-04-01...2019-04-01: 13.448 * 137.195978 = 1845.01

Vytápění:
  2018-04: 1.00 * 7790 * 0.09 = 701.10
  2018-05: 1.00 * 7790 * 0.02 = 155.80
  2018-06: 1.00 * 7790 * 0.0 = 0.00
  2018-07: 1.00 * 7790 * 0.0 = 0.00
  2018-08: 1.00 * 7790 * 0.0 = 0.00
  2018-09: 1.00 * 7790 * 0.01 = 77.90
  2018-10: 1.00 * 7790 * 0.08 = 623.20
  2018-11: 1.00 * 7790 * 0.14 = 1090.60
  2018-12: 1.00 * 7790 * 0.17 = 1324.30
  2019-01: 1.00 * 8820 * 0.19 = 1675.80
  2019-02: 1.00 * 8820 * 0.16 = 1411.20
  2019-03: 1.00 * 8820 * 0.14 = 1234.80

Ostatní poplatky:
  2018-04-01...2019-01-01: 9.00 * 716.00 = 6444.00
  2019-01-01...2019-04-01: 3.00 * 825.00 = 2475.00

Přijaté zálohy:
  brezen-unor: 12 * 4000 = 48000
